### ISSUE TITLE
Duplicate indexing

### ISSUE THREAD

**Issue by grigzy28** [iquidus/explorer#347]
1.7.3 version

![Screenshot_1](https://user-images.githubusercontent.com/30666305/74036930-dd900200-4982-11ea-85f9-67c221dfb9ae.png)

**Comment by grigzy28** [comment]
After some further looking into it.  

It is listing out the tx's as blocks in the main display(as you can see in the screenshot), but when you look at each transaction for a block it does have the block hash correct on the details page.

Why? Some setting somewhere?

**Comment by grigzy28** [comment]
I see now that it is listing Latest Transactions as the primary block explorer page, is there a way to get it to show Latest Blocks instead?

**Comment by uaktags** [comment]
Duplicate of https://github.com/iquidus/explorer/issues/238

TLDR: out of the box, there isnt a "latest blocks" view, because we don't record blocks as their own tracked data yet.

Closing
